Gemfile

```diff
@@ -230,7 +232,7 @@ end
 unless ENV["APPLIANCE"]
   group :development do
     gem "foreman"
-    gem "haml_lint",           "~>0.20.0", :require => false
+    gem "haml_lint",           "~>0.22.0", :require => false
```

Comment:
@himdel @h-kataria I'll need you to review the haml/haml_lint change needed for erubi.  Please see the relevant commit.